packages/grid/src/GridMetricCalculator.ts

```diff
@@ -241,19 +241,28 @@ class GridMetricCalculator {
       : 0;
     const treePaddingY = 0; // We don't support trees on columns (at least not yet)
 
-    const visibleRowHeights = new Map([
-      ...this.getVisibleRowHeights(state),
-      ...this.getFloatingRowHeights(state),
-    ]);
-    const visibleColumnWidths = new Map([
-      ...this.getVisibleColumnWidths(state, firstColumn, treePaddingX),
-      ...this.getFloatingColumnWidths(state),
-    ]);
+    let visibleRowHeights = this.getVisibleRowHeights(state);
+    let visibleColumnWidths = this.getVisibleColumnWidths(
+      state,
+      firstColumn,
+      treePaddingX
+    );
 
     // Calculate the metrics for the main grid
     const visibleRows = Array.from(visibleRowHeights.keys());
```

Comment:
The problem was the `visibleRows` map was including floatingRows which is wrong. I have a ticket to make things make more sense, but I figure making it make more sense is a breaking change so leaving it in another change is safest.